src/coreclr/pal/src/cruntime/printf.cpp

```diff
@@ -401,7 +401,7 @@ static BOOL Internal_ScanfExtractFormatW(LPCWSTR *Fmt, LPSTR Out, int iOutSize,
             {
                 ERROR("strcpy_s failed\n");
                 SetLastError(ERROR_INSUFFICIENT_BUFFER);
-                return FALSE;
+                goto EXIT;
```

Comment:
Shouldn't this set Result to FALSE before the goto?

Reply:
Result is initialized to FALSE: `BOOL Result = FALSE;`.